Packs/PaloAltoNetworks_IoT3rdParty/pack_metadata.json

```
@@ -0,0 +1,35 @@
+{
+    "name": "Palo Alto Networks IoT 3rd Party Integrations",
+    "description": "Palo Alto Networks IoT 3rd Party Integrations",
+    "support": "xsoar",
```

i think the support is community for this pack
```suggestion
    "support": "community",
```